style.css

```diff
@@ -0,0 +1,197 @@
+@font-face {
+  font-family: "Poppins";
+  src: url("https://fonts.googleapis.com/css?family=Poppins&display=swap");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  background-color: gray;
+  margin-left: 33vw;
+  margin-top: 3%;
+}
+
+body {
+  width: 462px;
+  height: 718px;
+  background: rgba(255, 255, 255, 1);
+  border-radius: 0 0 0 100px;
+  opacity: 1;
+  position: relative;
+  top: 0;
+  left: 0;
+  overflow: hidden;
+}
+
+header {
+  display: flex;
+  cursor: pointer;
+  width: 375px;
+  padding: 24px;
+  position: absolute;
+  top: 20px;
+  left: 34px;
+  margin: 12px;
+  z-index: 1;
+}
+
+.logo-text {
+  width: 76px;
+  color: #6070ff;
+  position: absolute;
+  top: 5px;
+  font-family: "Poppins", Arial, Verdana, Tahoma, sans-serif;
+  font-weight: bold;
+  font-size: 18px;
+  opacity: 1;
+  text-align: center;
+}
+
+.menu {
+  width: 48px;
+  height: 48px;
+  position: absolute;
+  top: 10px;
+  right: 20px;
+}
+
+.menu:hover {
+  transform: scale(1.2);
+}
+
+main {
+  width: 375px;
+  height: 604px;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
```

Comment:
- [ ] You have used background property here, But there is no background image. Please include that or remove these unnecessary code.